### DISCUSSION TITLE
v3.4 will not display watermarked file properly

### DISCUSSION THREAD

**Discussion by bobvance** [sumatrapdfreader/sumatrapdf#2617] [General]
Sumatra v3 works fine, as does Adobe Reader XI.
I have attached a screenshot of 3.3.3 vs. 3.4.2
and also the test file used.

[test_3.4_watermark.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/8791594/test_3.4_watermark.pdf)
![test_3 4_watermark_screen_capture](https://user-images.githubusercontent.com/29075259/170837325-fb320e34-06d5-4b0d-ae55-c2a8e144791b.PNG)
.

**Comment by kjk** [comment]
Fixed in https://github.com/sumatrapdfreader/sumatrapdf/commit/49d3f7ae45958726164dab7824287dc8ae0f21eb

Bugs are better reported as https://github.com/sumatrapdfreader/sumatrapdf/issues as I might forget about bug reported in a discussion
